webpack.icons.config.js

```diff
@@ -0,0 +1,54 @@
+const path = require("path");
+const {UglifyJsPlugin} = require('webpack').optimize;
+
+module.exports = {
+    entry: {
+        "clarity-icons-light.min": "./src/clarity-icons/index.ts",
+        "clarity-icons.min": "./src/clarity-icons/clarity-icons-sfx.ts",
+        "shapes/commerce-shapes": "./src/clarity-icons/shapes/commerce-shapes.ts",
+        "shapes/commerce-shapes.min": "./src/clarity-icons/shapes/commerce-shapes.ts",
+        "shapes/essential-shapes": "./src/clarity-icons/shapes/essential-shapes.ts",
+        "shapes/essential-shapes.min": "./src/clarity-icons/shapes/essential-shapes.ts",
+        "shapes/media-shapes": "./src/clarity-icons/shapes/media-shapes.ts",
+        "shapes/media-shapes.min": "./src/clarity-icons/shapes/media-shapes.ts",
+        "shapes/social-shapes": "./src/clarity-icons/shapes/social-shapes.ts",
+        "shapes/social-shapes.min": "./src/clarity-icons/shapes/social-shapes.ts",
+        "shapes/travel-shapes": "./src/clarity-icons/shapes/travel-shapes.ts",
+        "shapes/travel-shapes.min": "./src/clarity-icons/shapes/travel-shapes.ts",
+        "shapes/technology-shapes": "./src/clarity-icons/shapes/technology-shapes.ts",
+        "shapes/technology-shapes.min": "./src/clarity-icons/shapes/technology-shapes.ts"
+    },
+    output: {
+        path: path.resolve(__dirname, 'dist/clarity-icons'),
+        filename: '[name].js',
+        libraryTarget: 'umd'
```

Comment:
I think we need to also use `library` property to expose `ClarityIcons`.

Reply:
Thanks, I'll add this.

Reply:
Hi @Shijir did a quick test with this and seems like we actually shouldn't be adding the `library` property. Here's some relevant doc around it: https://webpack.js.org/configuration/output/#module-definition-systems

In particular:
```Note that omitting library will result in the assignment of all properties returned by the entry point be assigned directly to the root object, as documented under the object assignment section```

When we produce the package with the `library` defined, I'm seeing some errors. I suspect that this is because we expect `ClarityIcons` to be globally defined/available. The package produced without the `library` property is working as expected, in both importing through TypeScript or via inclusion of the script tags.